### PR TITLE
Ana/4399 add paywall

### LINKED ISSUE #4399: Add Paywall
Add Paywall for notifications.

Initially, registered users will automatically go premium and get access, so the paywall will just limit access to unregistered users.

There will be a Paywall.vue component that will follow Charlie's design: 

https://www.sketch.com/s/8ffba04e-5055-44dd-984c-c60a74f5103b/a/5g1gv7

The component shown here is already to be found in [this file](https://github.com/luniehq/lunie-marketing/blob/master/src/pages/Index.vue).

With Vue routes we will check: if the user is registered we send them to PageNotifications. Otherwise, we send them to the Paywall. The "Get Started" button in the Paywall will redirect (for now) to the SignInModal, so users can enter their email and get a magic link to sign up.

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4399 

**Description:**

~~Animation is not working for some reason :sweat_smile: ~~

Animation working! :rocket: 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/components/account/Paywall.vue [hunk 1]
```diff
@@ -0,0 +1,320 @@
+<template>
+  <div class="paywall-container">
+    <h3>
+      Lunie Notifications<br />
+      are here!
+    </h3>
+    <TmBtn
+      id="getStartedBtn"
+      value="Get Started"
```
Comment (faboweb) [inline app/src/components/account/Paywall.vue:9]: @jbibla This should say something about Premium

**Comment by Bitcoinera** [comment]
Mmm, maybe I should make it smaller though:

**Comment by Bitcoinera** [comment]
Ah, it is because in [Charlie's design](https://www.sketch.com/s/8ffba04e-5055-44dd-984c-c60a74f5103b/a/5g1gv7#Inspector) there is no left navbar :thinking:

**Comment by faboweb** [comment]
🚀 Deployed on https://5efe260a19bbcfc36cc7c9d8--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
How should I make the line seen in the design? I remember @jbibla doesn't like `<hr>` elements

**Comment by Bitcoinera** [comment]
Not looking too bad :v: 

image

**Comment by jbibla** [comment]
add an `overflow: hidden` so the notifications are hidden until they are in the window

File: app/src/components/common/Avatar.vue [hunk 2]
```diff
@@ -25,6 +25,7 @@ export default {
       let sprites = this.human ? humanSprites : regularSprites
       const options = { mood: [`happy`] }
       const avatars = new Avatars(sprites, options)
+      console.log(this.address)
```
Comment (jbibla) [inline app/src/components/common/Avatar.vue:28]: remove

Comment (Bitcoinera) [reply app/src/components/common/Avatar.vue:28]: oops, thank you

Comment (Bitcoinera) [reply app/src/components/common/Avatar.vue:28]: oops, thank you

**Comment by Bitcoinera** [comment]
> add an `overflow: hidden` so the notifications are hidden until they are in the window

I think it is already there, look at line 264 in the Paywall

**Comment by Bitcoinera** [comment]
Let's merge 💃 !

File: app/src/components/account/Paywall.vue (hunk 1 above)
Comment (jbibla) [reply app/src/components/account/Paywall.vue:9]: i think it's fine - after you click it you'll see this modal: 

Screen Shot 2020-07-06 at 8 01 12 PM

File: app/src/components/account/Paywall.vue [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+        title: `❓ Your validator just voted — are you happy with how they voted?`,
+      },
+      {
+        id: 12,
+        title: `Your validator went offline for 2000 blocks! 🔔`,
+      },
+    ],
+  }),
+  methods: {
+    handleIntercom() {
```
Comment (faboweb) [inline app/src/components/account/Paywall.vue:152]: Why do we need to handle Intercom here?

**Pull request merged**